lib/autocomplete-manager.coffee

```diff
@@ -110,26 +113,59 @@ class AutocompleteManager
     return if @isCurrentFileBlackListed()
     cursor = @editor.getLastCursor()
     return unless cursor?
-    cursorPosition = cursor.getBufferPosition()
-    currentScope = cursor.getScopeDescriptor()
-    return unless currentScope?
-    currentScopeChain = currentScope.getScopeChain()
-    return unless currentScopeChain?
-
-    options =
-      editor: @editor
-      buffer: @buffer
-      cursor: cursor
```

Comment:
If we do ultimately extend our function `@prefixForCursor(cursor)` to the provider, it would best be done in this function and the cursor would be required. Should we leave the cursor in?